common/components/Header/components/NavigationLink.jsx

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import translate from 'translations';
+import { Link } from 'react-router';
+import './NavigationLink.scss';
+
+type Props = {
+  link: {
+    name: string,
+    to?: string,
+    external?: boolean
+  },
+  location: Object
+};
+
+export default class NavigationLink extends React.Component {
+  props: Props;
+
+  render() {
+    const { link, location } = this.props;
+    const linkClasses = classnames({
```

Comment:
Cool, I threw it in there since we'd talked about it a little before. It's nothing too fancy, just really reduces the amount of:

```
const someConditionalClass = condition ? 'has-class' : '';
const otherConditionalClass = otherCond ? 'has-other-class' : '';

<div className={`MyClass ${someConditionalClass} ${otherConditionalClass`}>
```

to

```
const classes = classnames({
  'MyClass': true,
  'has-class': !!condition,
  'has-other-class': !!otherCondition
});

<div className={classes}>
```